ui/src/app/applications/components/application-status-panel/application-status-panel.scss

```diff
@@ -128,13 +134,15 @@
 
     &__item-value {
         display: flex;
-        align-items: center;
+        align-items: flex-end;
+        margin-bottom: 0.5em;
+        font-weight: 500;
```

Comment:
Is the designer a stickler in terms of how close the implementation is to the design?  It looks fine to me but if the designer wants to nitpick on the size of the font, have a look at this screenshot.  The left part is actual, the right part is the design from the issue:

<img width="574" alt="Screen Shot 2023-02-08 at 12 45 52 PM" src="https://user-images.githubusercontent.com/7726022/217611028-88e0ab8f-8c84-40ed-ab76-94d6c8129f7b.png">

Again, I'm ok with it.